### PR TITLE
PAYARA-3466 Update to Open API TCK to 1.1.2

### PR DESCRIPTION
**Pull request by jbee**
Needed update to 1.1 version of the TCK.

### CODE REVIEW

**Comment by jbee** [comment]
@MattGill98 I think we are in the process of looking into possible solutions to version dependent tests and modules. I didn't want to get side tracked by doing this on my own for this so I think we can and maybe should wait merging this until we have a general solution that we can apply here.

**Comment by jbee** [comment]
We should use 0.2 profile activator to only apply new OpenAPI versions to `payara.version`s that support it.

**Comment by jbee** [comment]
Added profiles to switch versions of API and TCK. The switched properties are also defined in `<properties>` block in order to have latest version available in IDE which makes debugging much easier. The profile will override this default.

File: MicroProfile-OpenAPI/tck-runner/pom.xml [hunk 1]
```diff
@@ -109,6 +109,32 @@
...
+                    <value>/4\..+|5\.18.+|5\.191.*/</value>
+                </property>
+            </activation>
+            <properties>
+                <microprofile.openapi.version>1.0.1</microprofile.openapi.version>
+                <microprofile.openapi.tck.version>1.0.1.payara-p1</microprofile.openapi.tck.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>mp-openapi-1.1</id>
```
Comment (MattGill98) [inline MicroProfile-OpenAPI/tck-runner/pom.xml]: This profile can probably be removed now right?

Comment (MattGill98) [reply MicroProfile-OpenAPI/tck-runner/pom.xml]: On second thoughts I like leaving this in for transparency

**Pull request merged**